src/wp-admin/includes/image-edit.php

```diff
@@ -293,25 +293,25 @@ function wp_image_editor( $post_id, $msg = false ) {
 		</div>
 		<div class="imgedit-thumbnail-preview-group">
 			<figure class="imgedit-thumbnail-preview">
-				<img src="<?php echo $thumb['url']; ?>" width="<?php echo $thumb_img[0]; ?>" height="<?php echo $thumb_img[1]; ?>" class="imgedit-size-preview" alt="" draggable="false" />
+				<img src="<?php echo esc_url($thumb['url']); ?>" width="<?php echo esc_attr($thumb_img[0]); ?>" height="<?php echo esc_attr($thumb_img[1]); ?>" class="imgedit-size-preview" alt="" draggable="false" />
```

Comment:
```suggestion
				<img src="<?php echo esc_url( $thumb['url'] ); ?>" width="<?php echo esc_attr( $thumb_img[0] ); ?>" height="<?php echo esc_attr( $thumb_img[1] ); ?>" class="imgedit-size-preview" alt="" draggable="false" />
```

Add spaces for coding standards.